packages/events-example/src/commands/reconcile/index.ts

```diff
@@ -123,10 +57,11 @@ export default class Reconcile extends Command {
     const { flags } = await this.parse(Reconcile);
 
     // pulumi setup
-    this.localWorkspace = await LocalWorkspace.create({
-      projectSettings: { name: flags.workspace, runtime: 'nodejs' },
-    });
-    this.localWorkspace.installPlugin('aws', 'v4.0.0');
+    this.deployment = await PulumiAwsDeployment.create(
+      "tryhard", 
+      { projectSettings: { name: flags.workspace, runtime: 'nodejs' } },
+      pulumiProgramGenerator
+    )
 
     // nile setup
     this.nile = Nile({ basePath: flags.basePath, workspace: flags.workspace });
```

Comment:
since we create our stacks inside `deployment`, does it make sense to also have a function to set up nile?
something like this is what I was thinking:
```
const { flags } = await this.parse(Reconcile);
this.initNile(flags);
const instances = await this.loadNileInstances(flags);
const latestSeq = findMaxSeq(instances);
const stacks = await this.deployment.loadStacks();
```

I think this would make it very clear where nile started and ended vs stack management

Reply:
yeah, I like this.

Reply:
New version is mostly like this.